src/test/java/co/omise/requests/SourceRequestTest.java

```diff
@@ -25,4 +26,23 @@ public void testCreate() throws IOException, OmiseException {
         assertEquals("redirect", source.getFlow().toString());
         assertEquals("alipay", source.getType().toString());
     }
+
+    @Test
+    public void testCreate_installment() throws IOException, OmiseException {
```

Comment:
Camel-case?

Reply:
But we do add `_` in the between for test methods right?